test/end-to-end/specs/api.test.js

```diff
@@ -0,0 +1,44 @@
+const commands = require("../utils/commands");
+
+// acceptable API response time
+// the 0.1 second comes from https://www.nngroup.com/articles/response-times-3-important-limits/
+const acceptable = 100;
```

Comment:
Great to have this kind of test. I'm a bit worried that this might become flaky on overloaded test servers though.

Reply:
Right, that's my worry as well. But we can see how it goes, and bump it if it fails noticeably on the infra.